python/orca/example/shard_keras_tutorial.py

```diff
@@ -37,7 +37,7 @@
 # compile the keras model
 model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])
 
-data_shard = transform_to_shard_dict(data_shard,
+data_shard = shardsPreprocessing(data_shard,
```

Comment:
"Preprocessing" is a very general term; be more specific about how the shards are transformed